test/Helper/UrlIntegrationTest.php

```diff
@@ -29,16 +27,15 @@
  */
 class UrlIntegrationTest extends TestCase
 {
+    private $serviceManager;
+
     protected function setUp(): void
     {
-        $this->literalRouteType = class_exists(V2HttpRoute\Literal::class)
```

Comment:
Similar to above: putting a restriction in `composer.json` is necessary for this conditional to be removed.